src/go/rpk/pkg/cli/cmd/group/describe.go

```diff
@@ -93,42 +97,45 @@ func printDescribed(
 	groups kadm.DescribedGroups,
 	fetched map[string]kadm.FetchOffsetsResponse,
 	listed kadm.ListedOffsets,
+	summary bool,
 ) {
 	for _, group := range groups.Sorted() {
 		lag := kadm.CalculateGroupLag(group, fetched[group.Group].Fetched, listed)
 
 		var rows []describeRow
 		var useInstanceID, useErr bool
-		for _, l := range lag.Sorted() {
-			row := describeRow{
-				topic:     l.End.Topic,
-				partition: l.End.Partition,
-
-				currentOffset: strconv.FormatInt(l.Commit.At, 10),
-				logEndOffset:  l.End.Offset,
-				lag:           strconv.FormatInt(l.Lag, 10),
-
-				err: l.Err,
-			}
-
-			if !l.IsEmpty() {
-				row.memberID = l.Member.MemberID
-				row.instanceID = l.Member.InstanceID
-				row.clientID = l.Member.ClientID
-				row.host = l.Member.ClientHost
+		if !summary {
```

Comment:
This can be moved above `CalculateGroupLag`, in fact we shouldn't even list offsets nor fetch offset commits. Easier and clearer would be:
- extract the summary information into a separate function
- in Run after DescribeGroups, if summary, call separate function and return.
This would allow us to avoid this deep conditional and extra tabbing, and ensures we avoid any extra unnecessary logic.